src/hooks/appData.ts

```diff
@@ -7,19 +7,31 @@ import { getApiHost, getData, getDataOrThrow } from '../config/utils';
 import { QueryClientConfig } from '../types';
 import { AppDataRecord } from '@graasp/sdk/frontend';
 import { convertJs } from '@graasp/sdk';
+import { WebsocketClient } from '../ws/ws-client';
+import { configureWsAppDataHooks } from '../ws/hooks/app';
 
-export default (queryClient: QueryClient, queryConfig: QueryClientConfig) => {
+export default (
+  queryClient: QueryClient,
+  queryConfig: QueryClientConfig,
+  websocketClient?: WebsocketClient,
+) => {
   const { retry, cacheTime, staleTime } = queryConfig;
   const defaultOptions = {
     retry,
     cacheTime,
     staleTime,
   };
+  const { useAppDataUpdates } = configureWsAppDataHooks(websocketClient);
   return {
-    useAppData: (refetchInterval: number | false = false) => {
+    useAppData: (options?: { refetchInterval: number | false; getUpdates?: boolean }) => {
+      const { refetchInterval, getUpdates } = options || { refetchInterval: false };
```

Comment:
Note that by destructuring it this way, `refetchInterval` does not get the `false` default value if the passed options only contain a value for `getUpdates`. Is this expected?

Example:
```ts
useAppData({ getUpdates: false })
// so the destructured values will be:
{ refetchInterval: undefined, getUpdates: false } <-- note that refetchInterval did not get the `false` default value
```